### DISCUSSION TITLE
Installation issue - singularity make error1

### DISCUSSION THREAD

**Discussion by lizi010922** [sylabs/singularity#2755] [Q&A]
Hello, I tried to install singularity, but there is an error.

The download is successful. But when I run the 'make' step, the error 
"Missing autoconf 
make: *** [Makefile:359: /singularity-ce-4.1.2/third_party/squashfuse/squashfuse_ll] error 1"  occured.

I followed every step in this official guide (https://docs.sylabs.io/guides/latest/admin-guide/index.html). 

I'm really confused , please tell me which step is wrong.

**Version of Singularity**
singularity-ce-4.1.2

$ go --version
go1.21.7

**Comment by dtrudg** [comment]
`autoconf` is included in the dependency installation step at...

https://docs.sylabs.io/guides/latest/admin-guide/admin_quickstart.html#install-dependencies

If you followed this step exactly, and it failed, you will need to provide more information about the whole process followed to install. We would need to know what Linux distribution you are using, and the exact commands that you ran.

As this is not a bug with Singularity, I will transfer the issue do a discussion Q&A item.

**Reply by WoCer2019** [reply]
Hi, my Linux distribution is Ubuntu.

No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.6 LTS
Release:	18.04
Codename:	bionic

I followed every step in this official guide (https://docs.sylabs.io/guides/latest/user-guide/quick_start.html#quick-installation-steps).

When I run 
sudo apt-get install -y \
   autoconf \
   automake \
   cryptsetup \
   git \
   libfuse-dev \
   libglib2.0-dev \
   libseccomp-dev \
   libtool \
   pkg-config \
   runc \
   squashfs-tools \
   squashfs-tools-ng \
   uidmap \
   wget \
   zlib1g-dev

echo "Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package squashfs-tools-ng"

How should I solve this problem? Thanks!
